src/main/fc/cli.c

```diff
@@ -648,17 +650,19 @@ static void printAux(uint8_t dumpMask, const modeActivationCondition_t *modeActi
                 && mac->auxChannelIndex == macDefault->auxChannelIndex
                 && mac->range.startStep == macDefault->range.startStep
                 && mac->range.endStep == macDefault->range.endStep;
+            const box_t *box = findBoxByActiveBoxId(macDefault->modeId);
             cliDefaultPrintLinef(dumpMask, equalsDefault, format,
                 i,
-                macDefault->modeId,
+                box->boxId,
                 macDefault->auxChannelIndex,
                 MODE_STEP_TO_CHANNEL_VALUE(macDefault->range.startStep),
                 MODE_STEP_TO_CHANNEL_VALUE(macDefault->range.endStep)
             );
         }
+        const box_t *box = findBoxByActiveBoxId(mac->modeId);
         cliDumpPrintLinef(dumpMask, equalsDefault, format,
             i,
-            mac->modeId,
+            box->boxId,
```

Comment:
likewise, permanentId 
isn't that the whole point of this patch set?

btw, attached is diff that should address all the comments in my reviews. This allows me to round-trip  `aux` with permanentId rather than boxId (which I think is an excellent idea btw).
[fixme-5654.diff.txt](https://github.com/iNavFlight/inav/files/4565462/fixme-5654.diff.txt)

Reply:
thanks you, i applied your patch